tools/ReleaseProcess/UploadArtifactsToGitHub.ps1

```diff
@@ -0,0 +1,459 @@
+## Copyright (c) Microsoft. All rights reserved.
+## Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+<#
+    .SYNOPSIS
+        This function downloads and prepares the Iot Edge Daemon artifacts from DevOps pipeline for a release. 
+        The final artifacts are collected under "<WorkDir>/output" path.
+    .DESCRIPTION
+        1) Download edgelet artifacts from a given BuildId from a DevOps pipeline
+        2) Extract the *.zip artifacts into their own directories
+        3) Traverse the directories and rename a relavant artifact into the following format: 
+              .DEB _________________ {component}_{version}_{os}_{arch}.deb 
+              .RPM _________________ {component}-{version}.{os}.{arch}.rpm  (No name change)
+        4) Move the renamed artifacts to a staging directory at "<WorkDir>/output"
+    .INPUTS 
+        $DevOpsPat ____________ (String) Environment variable contains DevOps Personal Access Token (with Build:Read privilege) in UTF8 encoding
```

Comment:
I think `.INPUTS` is about describing objects that are _piped to_ the script, not variables. So you probably shouldn't have an INPUTS section.

Reply:
[Link to docs](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comment_based_help?view=powershell-7.1#inputs) about comment-based help in PowerShell